BSB_lan_defs.h

```diff
@@ -7668,7 +7761,9 @@ PROGMEM_LATE const cmd_t cmdtbl2[]={
 {0x593D05A8,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8416,  STR8416,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Heissgastemperatur Max
 {0x593D05C0,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8417,  STR8417,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Heissgastemperatur 2
 {0x593D05C8,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8420,  STR8420,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Kältemitteltemperatur flüssig
-{0x053D163C,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8423,  STR8423,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Kondensatortemperatur
+{0x593D1783,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8423,  STR8423,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Kondensatortemperatur
+// todo: How to integrate this line? it's sent along with parameter 8423
+// {0x053D163C,  CAT_DIAG_ERZEUGER,    VT_PRESSURE_WORD, ????,  STR8423_1,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Kondensationsdruck
```

Comment:
Not sure which parameter number to assign. The datagram is sent and displayed along with parameter 8423.

Reply:
I'm not sure why you removed the 0x053D163C line. If it is a setting in the heater that displays two values under one parameter, then a new parameter number has to be created for one of the two values. As we have one user reporting that 0x053D163C is actually "Kondensatortemperatur", I would stick with this line and make 8324 with CommandID 0x593D1783 "Kondensatordruck"

Reply:
I just found this document describing that parameter 8423 actually consists of both values (as far as I understand it ). Is something like this yet possible?

http://www.mhg-schweiz.ch/de/downloads/aid!7b786239-40bf-4077-9a19-e2c97ebab6c5/id!185/

This also seems to apply to the other values in question.